RNTrueLayerPaymentsSDK/js/models/theme/AndroidColors.ts

```diff
@@ -0,0 +1,8 @@
+export type AndroidColors = {
+  primary?: string;
+  background?: string;
+  onBackground?: string;
+  surface?: string;
+  onSurface?: string;
+  error?: string;
+};
```

Comment:
maybe add documentation? Or the default values so if they run the app they know what is what

Reply:
Not sure if it makes sense to document the defaults here since it's based on Google's default material theme. We'll need to know whenever Google makes a change and update here every time.